R/svm.R

```diff
@@ -415,6 +415,25 @@ cuml_svm_regression_impl <- function(processed, cost, kernel, gamma, coef0,
   )
 }
 
+cuml_get_state.cuml_svr <- function(model) {
+  model_state <- list(
+    model_state = .svr_get_state(model$xptr),
```

Comment:
Current naming convention: anything starting with `.` is implemented in CUDA/C++.